apps/frontend/src/app/transformations-creator/transformations-creator.component.html

```diff
@@ -0,0 +1,45 @@
+<div class="container">
+  <div class="row">
+    <div class="col">
+      <form [formGroup]="transformationForm" (ngSubmit)="createTransformation()">
+        <div class="col mb-3">
+          <label for="name" class="form-label">Name:</label><br />
+          <input id="name" class="form-control" type="text" formControlName="name" />
+          <div *ngIf="transformationForm.controls['name'].invalid && transformationForm.controls['name'].touched" class="alert alert-danger">
```

Comment:
Something we often do, to make accessing the form controls for us easier, is to write a `get` function for it in the components typescript code. Will add an example suggestion there for you

Reply:
After defining the name control function in the typescript, it helps to easily access the form control.